src/plugins/intel_cpu/src/plugin.cpp

```diff
@@ -227,8 +232,41 @@ static std::string getDeviceFullName() {
     return brand_string;
 }
 
+// Print basic CPU information once when OV_CPU_VERBOSE=info is set
+static void print_plugin_info_once() {
+    static std::atomic_flag info_printed = ATOMIC_FLAG_INIT;
+    // @todo To avoid dealing with environment variables in multiple places
+    //       read them all here and pass to CPU config as a parameter
+    const auto& ov_cpu_verbose = ov::util::getenv_string("OV_CPU_VERBOSE");
+#if defined(CPU_DEBUG_CAPS)
+    // in case of debug caps enabled print info if OV_CPU_VERBOSE is set to any value
+    if (ov_cpu_verbose.empty()) {
+        return;
+    }
+#else
+    if (ov_cpu_verbose != "info") {
+        return;
+    }
+#endif
+
+    // Print info only once
+    if (info_printed.test_and_set()) {
+        return;
+    }
+
+    const auto& ov_version = ov::get_openvino_version();
+    const auto& dnnl_version = dnnl::version();
+    std::cout << "ov_cpu_verbose,info,intel_cpu_plugin,version: " << ov_version.buildNumber << '\n';
+    std::cout << "ov_cpu_verbose,info,onednn,version: " << dnnl_version->major << "." << dnnl_version->minor << "."
+              << dnnl_version->patch << " (commit " << dnnl_version->hash << ")" << '\n';
+    // @todo add more info regarding other backends if available
+    std::cout << "ov_cpu_verbose,info,isa: " << dnnl::impl::cpu::x64::get_isa_info() << '\n';
+    std::cout << "ov_cpu_verbose,info,intel_cpu_plugin,runtime: " << OV_THREAD_NAME << std::endl;
```

Comment:
It looks like more log message use, logger instead.
or it is used only for debug build?

Reply:
The idea is to do have it in release version.
So, for example, customer or CI pipeline jobs could use it to print useful information about environment without having to rebuild the product.

Reply:
Updated to use 'log_message"